sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -23,6 +23,7 @@
 
 from apache_beam.dataframe import expressions
 from apache_beam.dataframe import frame_base
+from apache_beam.dataframe import frames  # pylint: disable=unused-import
```

Comment:
What is this for?

Reply:
It makes sure the wrapper code is populated with the various types of frames.

Reply:
This ensures the deferred dataframe subclasses are registered for wrapping.